scripts/sanitizer-env.sh

```diff
@@ -5,7 +5,11 @@
 if [[ ${CC-} == *gcc* ]]; then
     OUR_PRELOAD=$("$CC" -print-file-name=libasan.so)
 elif [[ ${CC-} == *clang* ]]; then
-    OUR_PRELOAD=$("$CC" -print-file-name=libclang_rt.asan-x86_64.so)
+    # With v19, the library lacks the CPU arch in its name
+    OUR_PRELOAD=$("$CC" -print-file-name=libclang_rt.asan.so)
+    if ! test -f "$OUR_PRELOAD"; then
+       OUR_PRELOAD=$("$CC" -print-file-name=libclang_rt.asan-x86_64.so)
```

Comment:
humn... pre-existent issue:  I wonder if we should check arch instead of hardcoding x86_64

Reply:
Considering that clang is moving way from it, I'd rather avoid adding more complexity unless we get bug reports.